qutebrowser/config/configdata.yml

```diff
@@ -1764,6 +1764,14 @@ tabs.pinned.frozen:
     default: True
     desc: Force pinned tabs to stay at fixed URL.
 
+tabs.undo.stack_size:
```

Comment:
Hmm, `tabs.undo.stack_size` or `tabs.undo_stack_size`? No hard feelings either way, I guess if we assume that we add new `undo.*` options in the future, this is better.

Reply:
Either way, consistency with #4801 would be nice.

Reply:
Good point! Having a `tabs.focus.` group sounds a bit weird to me, so `tabs.undo_stack_size` would be better as it's consistent with `tabs.focus_stack_size` - unless we already plan more `tabs.undo.*` settings, then IMHO it's better to group those for consistency. I'll let @jgkamat decide.

Reply:
I think there will be more undo settings eventually, but I'll keep it `tabs.undo_stack_size` for simplicity at the moment. I don't think we'll ever have more focus settings